fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -123,7 +132,18 @@ Config parseConfiguration(final String[] args) {
             config.setRdfExtension(cmd.getOptionValue('x', DEFAULT_RDF_EXT));
             config.setRdfLanguage(cmd.getOptionValue('l', DEFAULT_RDF_LANG));
 
+            final String user = cmd.getOptionValue("user");
+            if (user != null) {
+                if (user.indexOf(':') == -1) {
+                    printHelp("user option must be in the format username:password");
+                } else {
+                    config.setUsername(user.substring(0, user.indexOf(':')));
+                    config.setPassword(user.substring(user.indexOf(':') + 1));
+                }
+            }
+
         } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
We don't want to be `e.printStackTrace()`ing. If you want this stack trace, turn on DEBUG: https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ImportExportDriver.java#L49